shared/common-adapters/clickable-box.desktop.js

```diff
@@ -68,7 +68,7 @@ class ClickableBox extends React.Component<Props & {children: any}, {mouseDown:
         onMouseLeave={needMouseEnterLeaveHandlers(this.props) ? this._onMouseLeave : undefined}
         onMouseUp={this._onMouseUp}
         onClick={onClick}
-        style={{..._containerStyle, ...(onClick ? globalStyles.clickable : null), ...style}}
+        style={collapseStyles([_containerStyle, onClick ? globalStyles.clickable : {}, style || {}])}
```

Comment:
you can keep null vs passing {} , this is desktop so you can spread null

Reply:
same for the style || {} thing, just use style

Reply:
On master `collapseStyles` is declared with `Array<Object>`, so I was getting errors. I'll switch it over once your changes are in